setup.py

```diff
@@ -22,6 +22,7 @@
     install_requires=[
         'clld>=4.6',
         'clldmpg~=3.1',
+        'clldutils>=2.8',
```

Comment:
I typically put packages that are required directly above packages which also have this requirement to make sure `pip` encounters the most specific req first. `clld` also requires `clldutils`, alas, `clld` 4.6 is happy also with older versions of `clldutils`. So I'd move `clldutils` to the top in this list.